containers.go

```diff
@@ -58,6 +59,7 @@ func ListContainers(client *gophercloud.ServiceClient, networks <-chan Resource)
 	}
 	go func() {
 		defer close(ch)
+		var gerr gophercloud.ErrDefault403
```

Comment:
We could perhaps use a more descriptive name for the variable, for instance errForbidden or something like that.

Reply:
done